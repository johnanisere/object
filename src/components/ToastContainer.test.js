
import React from "react";
import { shallow } from "enzyme";

import { ToastContainer } from "react-toastify";
import Toast from "./ToastContainer"

it("Toast component exists", () => {
    const wrapper = shallow(<Toast />);
    expect(wrapper.exists()).toBe(true);
});

it("Toast component renders", () => {
    const wrapper = shallow(<Toast />);
    expect(wrapper).toMatchSnapshot();
});

it(`Toast should contain "ToastContainer" `, () => {
    const wrapper = shallow(<Toast />);

    expect(wrapper.find(ToastContainer)).toHaveLength(1);
});