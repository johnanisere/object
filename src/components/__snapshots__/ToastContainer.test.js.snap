// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast component renders 1`] = `ShallowWrapper {}`;
